### DISCUSSION TITLE
Simple Resource Excessive Modal Queries

### DISCUSSION THREAD

**Discussion by Z3d0X** [filamentphp/filament#2445] [Help] [answered]
I've noticed when SimpleResource's Edit/Delete Modal open several queries are duplicated. It happens in all of the SimpleResources
FIlament Version: `v2.12.9`

Here is the screenshot from the debugbar.
![image](https://user-images.githubusercontent.com/75579178/168774768-ceae54e0-235d-432e-ab27-491ea5338dc9.png)

**Comment by danharrin** [comment, marked as answer]
Should be fixed now

**Reply by Z3d0X** [reply]
Fixed by https://github.com/laravel-filament/filament/pull/2537 and https://github.com/laravel-filament/filament/pull/2538
